### ISSUE TITLE
Add support for 'setup.py' nor 'pyproject.toml'

### ISSUE THREAD

**Issue by th0ma7** [morpheus65535/bazarr#1648]
Reference PR from SynoCommunity project: [#4998](https://github.com/SynoCommunity/spksrc/pull/4998)

Long-story short, instead of "manually" downloading from github I tried using pip with a requirement file such as:
```
# Download bazarr from github directly as not available on pypi
git+https://github.com/morpheus65535/bazarr@v1.0.2-beta.10#egg=bazarr

greenlet==1.1.2
# gevent==21.8.0              # requires to be built as cross/gevent
# numpy==1.21.4               # requires to be built as cross/numpy

# webrtcvad==2.0.10           # requires unreleased version at specific commit for qoriq arch
git+https://github.com/wiseman/py-webrtcvad@3bd761332a9404f5c9276105070ee814c4428342#egg=webrtcvad
```

But sadly it doesn't work:
```
Collecting bazarr
  Cloning https://github.com/morpheus65535/bazarr (to revision v1.0.2-beta.10) to /tmp/pip-wheel-4ijdoeiu/bazarr_4066a3b2c2a2435ba149f25aacd9d6ae
  Running command git clone --filter=blob:none -q https://github.com/morpheus65535/bazarr /tmp/pip-wheel-4ijdoeiu/bazarr_4066a3b2c2a2435ba149f25aacd9d6ae
  Running command git checkout -q 20eca6ade2edaffd36f695af30b4563f811a52fa
  Resolved https://github.com/morpheus65535/bazarr to commit 20eca6ade2edaffd36f695af30b4563f811a52fa
ERROR: bazarr from git+https://github.com/morpheus65535/bazarr@v1.0.2-beta.10#egg=bazarr (from -r /home/spksrc/python38-others/spksrc/spk/bazarr/work-qoriq-6.1/wheelhouse/requirements-crossenv.txt (line 1)) does not appear to be a Python project: neither 'setup.py' nor 'pyproject.toml' found.
```

Would be great if that support was added, cheers!

**Comment by morpheus65535** [comment]
That's a feature request, not an issue. It's never been in the plan that Bazarr could be deployed as any other python module: it's not a module.

Anyway, installing using git is unsupported and you'll need to deploy the dev environment to build frontend. That's the reason why the instructions on the wiki tell to use the bazarr.zip asset attached to the release.

**Comment by th0ma7** [comment]
Understood and thnx for the quick reply.  Cheers & Happy Holiday!

**Comment by th0ma7** [comment]
@morpheus65535 I've been working on python 3.10 integration for many of our SynoCommunity packages and as such I've played with a few pieces of software.  Yesterday I noticed that the `salt` project is sort of similar to your, which really isn't a module but do work with `pip install`.  I thought this might be of interest if you ever want to further investigate this route as this could serve as an example.

For reference:
- https://repo.saltproject.io/
- https://github.com/SynoCommunity/spksrc/pull/5017

Cheers again!
